docs/observation/payload.md

```diff
@@ -0,0 +1,43 @@
+# Format of observation payload
+
+Notifications received by OneDrive have a format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the `onedrive-change-listener`, they get converted to a portable format before sent to the next stage, e.g.:
+
+```
+{
+  "name": "me--myrepo--master",
+  "params": {
+    "type": "onedrive",
+    "driveId": "...",
```

Comment:
`driveId` is specific to `onedrive`, where as github and gdocs have different identifiers.... maybe have a distinct `sourceProvider` or `provider` object...